### ISSUE TITLE
scrcpy Not working on Windows XP SP3 32-bit

### ISSUE THREAD

**Issue by SuckyBuckyYT** [Genymobile/scrcpy#4069]
well as the title says screen copy doesnt work on windows xp sp3 32 bit ive used 2.0 and the error that was given was "The procedure entry point DeleteProcThreadAtrributeList could not be located in the dynamic link library KERNEL32.DLL"and i tried using version 1.2 too but it gave me the error "This application has failed to start because bcrypt.dll was not found. Re-installing the application may fix this problem"

**Comment by rom1v** [comment]
Yes, Windows XP is probably not supported by scrcpy dependencies.
